cmd/appeal.go

```diff
@@ -53,6 +53,9 @@ func listAppealsCommand(c *app.CLIConfig) *cobra.Command {
 			$ guardian appeal list --role=viewer
 		`),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			s := term.Spin("Fetching appeal list")
+			defer s.Stop()
```

Comment:
@ishanarya0 Do we need this defer if we are closing spin below explicitly anyway.

Reply:
Yes. If an error is returned at line 64 or line 74, then we won't be able to close the spinner on line 80.

Reply:
The spinner label and frozen spinner will remain on the terminal, followed by the error message. In order to remove it, we need to close the spinner. Or, if we are okay with this state, then we can remove defer.

Example: https://github.com/odpf/guardian/pull/117#issuecomment-1016250527